### PR TITLE
Services and auth rework

### PR DESCRIPTION
**Pull request by ktsakalozos**
This is more affirmative than the previous version and more clear on what is used and how.

### CODE REVIEW

**Comment by ktsakalozos** [comment]
@evilnick @tvansteenburgh please have a look. Thanks

File: docs/ports.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
 ---
 layout: docs
-title: "Services exposed and ports used"
----
+title: "Services exposed - Authorization and authentication"
+---------------------------------------------------------------------
```
Comment (nottrobin) [inline docs/ports.md]: Could you leave this as 3 dashes?

Comment (pmahnke) [reply docs/ports.md]: it is a requirement of front-matter

File: docs/ports.md [hunk 2]
```diff
@@ -42,24 +42,29 @@ Both these services are exposed through unix sockets.
...
 
-Authentication of users is done with a [Static Password File](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authentication-strategies) also generated at first MicroK8s deployment. Password tokens and usernames are stored in the `basic_token.csv` file available under `/var/snap/microk8s/current/credentials/`. Also under `/var/snap/microk8s/current/credentials/` you can find the `client.config` kubeconfig file used by `microk8s.kubectl`.
+The authentication [strategies](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authentication-strategies)
+ enabled by default are:
+ - Static Password File, with password tokens and usernames are stored in `/var/snap/microk8s/current/credentials/basic_token.csv`
```
Comment (tvansteenburgh) [inline docs/ports.md]: s/are//

**Comment by ktsakalozos** [comment]
Any update on this?

**Comment by nottrobin** [comment]
Looks fine to me from a format perspective. I can't speak to the validity of the wording.

**Comment by ktsakalozos** [comment]
Do we have an ETA on when this would land? We released the new version of MicroK8s a couple of days ago.

**Comment by nottrobin** [comment]
@ktsakalozos it can land as soon as it's approved. I guess @tvansteenburgh is now happy after the typo fix? I can land it now if you like?

**Comment by nottrobin** [comment]
I just released this: https://microk8s.io/docs/ports

**Pull request merged**